apps/web/lib/event-types/getServerSideProps.tsx

```diff
@@ -1,13 +1,18 @@
 import type { GetServerSidePropsContext } from "next";
 
 import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
+import { ENABLE_INFINITE_EVENT_TYPES_FOR_ORG } from "@calcom/lib/constants";
 
 import { ssrInit } from "@server/lib/ssr";
 
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
   const ssr = await ssrInit(context);
   const session = await getServerSession({ req: context.req, res: context.res });
 
+  const isInfiniteScrollEnabled = session?.user?.org?.slug
+    ? ENABLE_INFINITE_EVENT_TYPES_FOR_ORG.includes(session.user.org.slug)
+    : false;
+
```

Comment:
Could be moved below session `nullish` check. The condition would need two less ?s